docs/en/enterprise-edition/policy-reference/sast-policies/python-policies/sast-policy-175.adoc

```diff
@@ -57,14 +57,14 @@ Secure code example:
 [source,python]
 ----
 from flask import request, render_template
-import cgi
+import html
 
 @app.route('/example')
 def example():
-    user_input = cgi.escape(request.args.get('user_input'))
+    user_input = html.escape(request.args.get('user_input'))
     return render_template('example.html', input=user_input)
 ----
 
-In the Improved code, the user's input goes through the 'cgi.escape()' function which neutralizes any potential script-related HTML tags. Thus, it makes it safe to include in the rendered web page.
+In the improved code, the user's input goes through the 'html.escape()' function which neutralizes any potential script-related HTML tags. Thus, it makes it safe to include in the rendered web page.  'cgi.escape()' will also pass but note that it has been deprecated in favour of 'html.escape()'.
```

Comment:
In the code fix, the user's input is now processed through the 'html.escape()' function, mitigating any risks associated with script-related HTML tags. This ensures the safety of the input when included in the rendered web page. 
NOTE: 'cgi.escape()' is still functional, but it has been deprecated in favor of 'html.escape()' for improved security practices.

Reply:
This all looks correct